src/it/java/teammates/it/sqllogic/core/UsersLogicIT.java

```diff
@@ -0,0 +1,114 @@
+package teammates.it.sqllogic.core;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.InstructorPermissionRole;
+import teammates.common.datatransfer.InstructorPrivileges;
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.Const;
+import teammates.it.test.BaseTestCaseWithSqlDatabaseAccess;
+import teammates.sqllogic.core.AccountsLogic;
+import teammates.sqllogic.core.CoursesLogic;
+import teammates.sqllogic.core.UsersLogic;
+import teammates.storage.sqlentity.Account;
+import teammates.storage.sqlentity.Course;
+import teammates.storage.sqlentity.Instructor;
+import teammates.storage.sqlentity.Student;
+
+/**
+ * SUT: {@link UsersLogic}.
+ */
+public class UsersLogicIT extends BaseTestCaseWithSqlDatabaseAccess {
```

Comment:
The test I really need to see here is the one that shows that deleting orphaned accounts works as expected. We cannot afford for this to have bugs.